src/components/TransactionHistoryRows/TransactionDetails/TransferColumn.tsx

```diff
@@ -36,10 +38,32 @@ export const TransferColumn = (transfer: TransferColumnProps) => {
       py={2}
     >
       <Row title='from' justifyContent='flex-start' flexDirection='column' alignItems='flex-start'>
-        <Address explorerAddressLink={transfer.asset.explorerAddressLink} address={transfer.from} />
+        {transfer.from.length ? (
+          (transfer.from as string[]).map(address => (
+            <Box key={address}>
+              <Address explorerAddressLink={transfer.asset.explorerAddressLink} address={address} />
+            </Box>
+          ))
+        ) : (
+          <Address
+            explorerAddressLink={transfer.asset.explorerAddressLink}
+            address={transfer.from as string}
+          />
+        )}
       </Row>
       <Row title='to' justifyContent='flex-start' flexDirection='column' alignItems='flex-start'>
-        <Address explorerAddressLink={transfer.asset.explorerAddressLink} address={transfer.to} />
+        {transfer.to.length ? (
+          (transfer.to as string[]).map(address => (
```

Comment:
Casting will need to be removed before opening for obvious reasons

Reply:
yep, got my todo in there for cleanup around this (touches all chain adapters which I didn't want to deal with right away haha)